geopandas/array.py

```diff
@@ -296,6 +296,37 @@ def sindex(self):
             self._sindex = get_sindex_class()(self.data)
         return self._sindex
 
+    def sindex_generated(self):
+        """Check the existence of the spatial index without generating it.
+
+        Use the `.sindex` attribute on a GeoDataFrame or GeoSeries
+        to generate a spatial index if it does not yet exist,
+        which may take considerable time based on the underlying index
+        implementation.
+
+        Note that the underlying spatial index may not be fully
+        initialized until the first use.
+
+        Examples
```

Comment:
The examples are based on this function at the `GeoDataFrame` level, not `GeometryArray` level.  I suggest removing them from here, and putting in a cross reference instead:

```
See Also
---------
GeoDataFrame.sindex_generated()
```

Reply:
Done